test/datadog/lambda.spec.rb

```diff
@@ -92,10 +92,10 @@
       expect(Datadog::Lambda.gen_enhanced_tags(ctxa)).to include(
         { account_id: '172597598159',
           cold_start: false,
-          functionname: 'ruby-test',
+          functionname: 'Ruby-test',
```

Comment:
I'm curious, why is "Ruby" capitalized here?

Reply:
These are the enhanced metrics tags. The change only applies to Lambda span metadata tags.

Reply:
For the enhanced metrics tags, the intake takes care of the lowercasing.